ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,243 @@
+---
+eip: xxxx
+title: JSON-RPC API for RIP-7560 Native Account Abstraction
+description: A set of JSON-RPC API methods that are required for a fully functional Native Account Abstraction protocol
+author:
+discussions-to:
+status: Draft
+type: Standards Track
+category: ERC
+created:
+requires: RIP-7560, ERC-7562, EIP-7702
+---
+
+## Abstract
+
+[RIP-7560](https://github.com/ethereum/RIPs/blob/master/RIPS/rip-7560.md) defines the new transaction type
+and the modifications to the EVM needed for the Native Account Abstraction support.
+
+However, there are a number of modifications to the Ethereum JSON-RPC API that is needed as well.
+This proposal contains the full description of the new or modified APIs, and it would be highly beneficial
+for the Native Account Abstraction ecosystem to implement these APIs in a standardised and compatible way.
+
+## Motivation
+
+Native Account Abstraction is expected to supersede [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337)
```

Comment:
I try not to use terms like "supersede".  Native AA and 4337 are going to live alongside each other for a long time, and possibly forever if L1 doesn't adopt native AA.  "supersede" might create the wrong impression that 4337 is going to be deprecated - which is not the case.  How about "improve upon"?